### DISCUSSION TITLE
How Do I Run mods on EzQuake

### DISCUSSION THREAD

**Discussion by M0ldyBread** [ezQuake/ezquake-source#841] [General]
I've been trying to run mods on EzQuake flathub and I don't know the command to run files.

**Comment by Calinou** [comment]
Note that most modern singleplayer mods (such as Arcane Dimensions) don't work with ezQuake, as they expect a NetQuake-based source port and not a QuakeWorld-based one. You need to use Ironwail or QuakeSpasm-Spiked for these.

**Comment by lucaslndfhp493-commits** [comment]
Die meisten modernen Singleplayer-Mods wie Arcane Dimensions funktionieren nicht mit ezQuake, da sie einen NetQuake-basierten Sourceport erwarten und keinen QuakeWorld-basierten. Um diese Mods zu https://robocatcasino-de.de/ spielen, sollte man Ironwail oder QuakeSpasm-Spiked verwenden, da diese die erforderliche Kompatibilität bieten. Nur so laufen die Mods stabil und mit allen Features, die für neuere Singleplayer-Erweiterungen nötig sind.

**Comment by OliverWatson373** [comment]
Se stai usando ezQuake installato da Flathub, il punto è che gira in sandbox, quindi non puoi semplicemente lanciare file come su una versione normale. Di solito devi mettere i mod nella cartella giusta, tipo dentro “.var/app/.../id1” o creare una directory dedicata al mod. Poi avvii il gioco con un comando come “-game nomemod” passando il nome della cartella. In [cleobetracasinos.it](https://cleobetracasinos.it/) alternativa puoi usare flatpak run con parametri, ad esempio aggiungendo --command, ma spesso è più semplice configurarlo da dentro il gioco stesso.
